python/metashape_workflow.py

```diff
@@ -7,11 +7,13 @@
 
 import argparse
 import sys
+from pathlib import Path
 
 # ---- If this is a first run from the standalone python module, need to copy the license file from the full metashape install: from python import metashape_license_setup
 
 ## Define where to get the config file (only used if running interactively)
```

Comment:
Is this comment and the other one about running interactively outdated at this point?